sources/Outputs/src/ParGenerator.cpp

```diff
@@ -248,17 +254,18 @@ ParGenerator::writeGenerator(const algo::GeneratorDefinition& def, const std::st
   // Qmax and QMin are determined in dynawo according to reactive capabilities curves and min max
   // we need a small numerical tolerance in case the starting point of the reactive injection is exactly
   // on the limit of the reactive capability curve
-  if (def.hasTransformer() && (def.isUsingRectangularDiagram())) {
+  if ((def.hasTransformer() && (def.isUsingRectangularDiagram())) || def.hasRpcl()) {
```

Comment:
I don't get this condition here. If has Rpcl and isUsingDiagram we shouldn't have QMin/QMax but QMin0/QMax0 no ?

Reply:
the diagram here is rectangular, so no need here to use a complex diagram (that means no need of a combiTable and QMin0/QMax0)

Reply:
My bad, thanks !

Reply:
If it can help: Those parameters are needed with model GeneratorPVRpclSignalN

Reply:
does it looks better now?